src/cbmc/cbmc_parse_options.cpp

```diff
@@ -662,6 +662,8 @@ int cbmc_parse_optionst::get_goto_program(
     if(!binaries.empty())
       config.set_from_symbol_table(symbol_table);
 
+    config.set_addr_bits(cmdline, symbol_table);
```

Comment:
You can't set language defaults at this point because the language isn't known yet.